ejercicio1-html-y-css/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="styles.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Login</title>
+</head>
+<body>
+    <div class="form">
+    <form>
+        <h1>Login</h1>
+        <label for = "email"> Email </label><br>
```

Comment:
Acá estos key value de los atributos no deberían de tener espacios,
`<label for="email">`.

Igualmente, todos los <br> que tienes puedes modificarlos por un margin entre los elementos.